content/teknologi/altinnstudio/altinn-api/models/app-metadata/_index.md

```diff
@@ -76,6 +76,20 @@ id | An id or key to identify the specific rule.
 path | A path to a specific field or property in the form model. 
 dataTypeId | The name of the datatype. See [DataType](#datatype). 
 
+## DataField
+
```

Comment:
missing entry in the table ca line 25

Reply:
Nice catch, I'm too used to automatic generated indexes:)